en/4-blocks/1-Introduction_to_blocks_and_blockchain.md

```diff
@@ -0,0 +1,12 @@
+# Introduction to blocks & blockchain
+
+## Concept
+When Alice wants to send a transaction to Bob, she will sign her transaction and broadcast it to the nodes in the network. The consensus nodes will select transactions that they will finalise, and include them into a block. A block is a batch of transactions that get confirmed and stored at the same time. Apart from all transactions that a certain block entails, there are some other fields present in each block which are discussed [here](2-Structure_of_a_block.md).
+
+## Chaining the blocks
+One of these fields in each block (b<sub>n</sub>) is a reference to the previous block that was just confirmed before this block (b<sub>n-1</sub>). In more detail, is is the hash of the header of the previous block. It is up to the nodes to calculate the hash to validate the parent. In its turn, this previous block(b<sub>n-1</sub>) has the same behaviour, linking to the block before it(b<sub>n-2</sub>). This chain goes all the way back to the first block ever created, called the Genesis (b<sub>0</sub>) block. This is where the term 'blockchain' comes from, as it is, in fact, a chain of blocks.
```

Comment:
is is the hash of the header of the previous block ->
it is the hash of the header of the previous block

Reply:
fixed, will push with other fixes.